api/handlers/export/relatorios.py

```
@@ -0,0 +1,210 @@
+            status_code=500,
+            detail={
+                "error": "PDF generation failed",
+                "message": "Erro ao gerar relatório PDF. Tente novamente."
+            }
+        )
+
+
+@router.post("/exportar-relatorio-pdf-completo")
+@limiter.limit("5/minute")
+async def exportar_relatorio_pdf_completo(
+    request: Request,
```

Ruff (0.14.5)

Unused function argument: `request`

(ARG001)